frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelMetrics.cy.ts

```diff
@@ -369,18 +378,178 @@ describe('Model Metrics', () => {
     modelMetricsPerformance.visit('test-project', 'test-inference-service');
     modelMetricsBias.findTab().should('not.exist');
   });
-  it('Enable Trusty AI Checkbox Available', () => {
+  it('Disable Trusty AI', () => {
+    initIntercepts({
+      disableBiasMetrics: false,
+      disablePerformanceMetrics: false,
+      hasServingData: false,
+      hasBiasData: false,
+    });
+
+    projectDetailsSettingsTab.visit('test-project');
+    projectDetailsSettingsTab
+      .findTrustyAIInstallCheckbox()
+      .should('be.enabled')
+      .should('be.checked');
+
+    // test disabling
+    cy.intercept(
+      {
+        method: 'DELETE',
+        pathname:
+          '/api/k8s/apis/trustyai.opendatahub.io.trustyai.opendatahub.io/v1alpha1/namespaces/test-project/trustyaiservices/trustyai-service',
+      },
+      {},
+    ).as('uninstallTrustyAI');
+
+    projectDetailsSettingsTab.findTrustyAIInstallCheckbox().uncheck();
+    projectDetailsSettingsTab
+      .getTrustyAIUninstallModal()
+      .findSubmitButton()
+      .should('not.be.enabled');
+    projectDetailsSettingsTab.getTrustyAIUninstallModal().findInput().type('trustyai');
+    projectDetailsSettingsTab
+      .getTrustyAIUninstallModal()
+      .findSubmitButton()
+      .should('be.enabled')
+      .click();
+
+    cy.wait('@uninstallTrustyAI').then((interception) => {
+      expect(interception.request).to.exist;
+    });
+  });
+  it('Enable Trusty AI', () => {
     initIntercepts({
       disableBiasMetrics: false,
       disablePerformanceMetrics: false,
       hasServingData: false,
       hasBiasData: false,
+      isTrustyAIInstalled: false,
     });
 
     projectDetailsSettingsTab.visit('test-project');
-    projectDetailsSettingsTab.findTrustyAIInstallCheckbox().should('be.enabled');
+    projectDetailsSettingsTab
+      .findTrustyAIInstallCheckbox()
+      .should('be.enabled')
+      .should('not.be.checked');
+
+    // test enabling
+    cy.intercept(
+      {
+        method: 'POST',
+        pathname:
+          '/api/k8s/apis/trustyai.opendatahub.io.trustyai.opendatahub.io/v1alpha1/namespaces/test-project/trustyaiservices',
+      },
+      mockTrustyAIServiceK8sResource({ isAvailable: true }),
+    ).as('installTrustyAI');
+
+    cy.intercept(
+      {
+        method: 'GET',
+        pathname:
+          '/api/k8s/apis/trustyai.opendatahub.io.trustyai.opendatahub.io/v1alpha1/namespaces/test-project/trustyaiservices/trustyai-service',
+      },
+      mockTrustyAIServiceK8sResource({
+        isAvailable: false,
+      }),
+    ).as('getTrustyAILoading');
+
+    projectDetailsSettingsTab.findTrustyAIInstallCheckbox().check();
+
+    cy.wait('@installTrustyAI').then((interception) => {
+      expect(interception.request).to.exist;
+    });
```

Comment:
There should be at least one test that asserts the expected payload for the request.

Reply:
done